build_resolvers/lib/src/resolver.dart

```diff
@@ -17,6 +17,8 @@ import 'package:analyzer/error/error.dart';
 import 'package:analyzer/file_system/physical_file_system.dart';
 // ignore: implementation_imports
 import 'package:analyzer/src/clients/build_resolvers/build_resolvers.dart';
+// ignore: implementation_imports
+import 'package:analyzer/src/dart/sdk/sdk.dart';
```

Comment:
I have a feeling the other library in the analyzer package has only been added so that we can avoid implementing other internal libraries here? Should a method to get a list of SDK library URIs be added to the `clients/build_resolvers` library then?

Reply:
cc @scheglov

Reply:
I've migrated to the new API, but I'll mark this PR as WIP until the analyzer version is released.

Reply:
SGTM, we will also need to increase the min analyzer version in the pubspec (not sure if we know the number yet, so you may have to wait)